README.md

```diff
@@ -250,20 +250,17 @@ jobs:
       group: ${{ github.workflow }}-${{ github.ref }}
     steps:
       - uses: actions/checkout@v3
-        with:
-          submodules: recursive  # Fetch the Docsy theme
-          fetch-depth: 0         # Fetch all history for .GitInfo and .Lastmod
```

Comment:
I think we still need the fetch-depth.

Reply:
> I think we still need the fetch-depth.

Yes, you are right. Line is added again.